### DISCUSSION TITLE
CM4 Carrier Wishlist

### DISCUSSION THREAD

**Discussion by dwery** [geerlingguy/raspberry-pi-pcie-devices#167] [General]
This repo seems like the best place where to track wishes, so... here’s mine:

- 6/8 USB2, NVMe and Gb Ethernet
- Small form factor, 1 USB, Gb Eth, no frills, integrated 4xSATA AHCI controller

**Comment by jamesy0ung** [comment]
Here's mine

- Exact same form factor as Raspberry Pi 3, so I can upgrade my pi-top

**Comment by geerlingguy** [comment]
Going to move this over to a discussion since it's not really an actionable issue (at least as it pertains to the Pi PCIe site).
